ocean-components/src/main/java/br/com/useblu/oceands/components/compose/OceanAccordion.kt

```diff
@@ -90,17 +92,20 @@ fun OceanAccordion(
 
         AnimatedVisibility(visible = expandedContent) {
             Column {
-                OceanSpacing.StackXXS()
-
                 Text(
                     text = description,
                     color = OceanColors.interfaceDarkDown,
                     fontSize = OceanFontSize.xxs,
                     modifier = Modifier.fillMaxWidth()
                 )
+                OceanSpacing.StackXS()
             }
-
         }
+
+        Divider(
```

Comment:
se esse for o divider entre Acorddions diferentes, acho melhor fazer o tratamento dentro do forEach ou do LazyColumn que chamar os Accordions, pq senao o ultimo da lista vai sempre ter um divider sobrando.
la na lazycolumn ou forEach da pra pegar o indice do item atual e caso for o ultimo nao exibe o divider

Reply:
Opa, @fernandozanutto esse divider ele aparece mesmo se houver um Accordion no fim de uma lista. Se você olhar ali nas definições  do componente, o Divider sempre está lá.

Reply:
ah top entao